### PR TITLE
Add fast deceleration logic

### PR DESCRIPTION
**Pull request by Tom-Willemsen**
https://github.com/ISISComputingGroup/IBEX/issues/5735

This can't reasonably be tested using the IOC test framework as we would need to fully emulate galil code.

### CODE REVIEW

File: GalilSup/Db/galil_sans2d_front_det_limit_footer.gmc [hunk 1]
```diff
@@ -1,4 +1,12 @@
 NO*************FOOTER*********************
+IF (fastStopFB=1)
+    oldecelC=_DCC;DCC=67107840;DCC=oldecelC;STC;
```
Comment (DominicOram) [inline GalilSup/Db/galil_sans2d_front_det_limit_footer.gmc:3]: If you're changing acceleration back before doing the stop it's not doing anything presumably? Where does the number 67107840 come from? Questions also apply to limit switch code below.

File: GalilSup/Db/galil_sans2d_rear_det_limit_footer.gmc [hunk 2]
```diff
@@ -1,4 +1,12 @@
 NO*************FOOTER*********************
+IF (fastStopRD=1)
+    oldecelA=_DCA;DCA=(_DCA*10);STA;DCA=oldecelA;
+    fastStopRD=0
+ENDIF
+IF (fastStopRB=1)
+    oldecelH=_DCH;DCH=67107840;DCH=oldecelH;STH;
```
Comment (DominicOram) [inline GalilSup/Db/galil_sans2d_rear_det_limit_footer.gmc:7]: See above comment for FB

**Review by DominicOram — CHANGES_REQUESTED** [review]
I'm not sure what the behaviour of sending a `DC*` during a move is, the manual says it's only valid to do during a jog. I'm concerned that either:
* The axis goes back to a slow deceleration before it's stopped
* The axis refuses to set the old deceleration

However, I don't think waiting until the move is finished to set the deceleration back is good because it will hang the thread, meaning we can't do the other fast stop. I think this may be something to look at on the hardware.

File: GalilSup/Db/galil_sans2d_front_det_limit_footer.gmc (hunk 1 above)
Comment (Tom-Willemsen) [reply GalilSup/Db/galil_sans2d_front_det_limit_footer.gmc]: `67107840` was the number used for the fast deceleration code under SECI in `LabVIEW Modules\Instruments\SANS2D\Sans2d Galil`. I don't know what it corresponds to physically (probably max acceleration that galil allows?) but I decided to make the behaviour of a collision avoidance stop consistent bump strips

**Review by DominicOram — APPROVED** [review]

**Pull request merged**